neurodsp/tests/test_timefrequency_hilbert.py

```diff
@@ -1,42 +1,83 @@
 """Test functions for time-frequency Hilbert analyses."""
 
-from neurodsp.tests.settings import FS
+import numpy as np
+
+from neurodsp.tests.settings import FS, N_SECONDS, EPS
 
 from neurodsp.timefrequency.hilbert import *
 
+from neurodsp.utils.data import create_times
+
 ###################################################################################################
 ###################################################################################################
 
-def test_robust_hilbert():
+def test_robust_hilbert(tsig_sine):
```

Comment:
I didn't really step through and refactor this test func like I did with the phase / amp / freq ones below. Could you check the updates below, and see if there are any analogous updates for this function?